Dockerfile

```diff
@@ -17,7 +19,7 @@ RUN license-check -path ./ --verbose=false \
  && go test $(go list ./... | grep -v /vendor/ | grep -v /template/|grep -v /build/) -cover \
  && VERSION=$(git describe --all --exact-match `git rev-parse HEAD` | grep tags | sed 's/tags\///') \
  && GIT_COMMIT=$(git rev-list -1 HEAD) \
- && CGO_ENABLED=0 GOOS=linux go build --ldflags "-s -w -X github.com/openfaas/faas-cli/version.GitCommit=${GIT_COMMIT} -X github.com/openfaas/faas-cli/version.Version=${VERSION}" -a -installsuffix cgo -o faas-cli
```

Comment:
CGO=1 for docker credentials helper

Reply:
Interesting - where are we importing the credentials helper?

Reply:
in the push operation (exec.go : Push())